### ISSUE TITLE
Finalise VQA Approach

### ISSUE THREAD

**Issue by hauntedcupoftea** [projectayre/ayre#11]
WORK ON THIS AFTER SENTIMENT IS COMPLETE.

**Comment by hauntedcupoftea** [comment]
For this proof of concept, we have appended custom pixel masks as image attention masks on a pretrained model. Further fine-tuning is needed, of course, but as a proof of concept it's output does not seem to be worse
